package.json

```diff
@@ -41,16 +43,49 @@
 		"eventemitter3": "^3.1.0"
 	},
 	"devDependencies": {
-		"@types/node": "^11.13.0",
+		"@sindresorhus/tsconfig": "^0.3.0",
+		"@types/benchmark": "^1.0.31",
+		"@types/in-range": "^1.0.0",
+		"@types/node": "^11.13.2",
+		"@types/random-int": "^1.0.0",
+		"@typescript-eslint/eslint-plugin": "^1.6.0",
 		"ava": "^1.4.1",
 		"benchmark": "^2.1.4",
 		"codecov": "^3.3.0",
+		"del-cli": "^1.1.0",
 		"delay": "^4.1.0",
+		"eslint-config-xo-typescript": "^0.9.0",
 		"in-range": "^1.0.0",
 		"nyc": "^13.3.0",
 		"random-int": "^1.0.0",
-		"time-span": "^2.0.0",
-		"tsd": "^0.7.2",
+		"time-span": "^3.1.0",
+		"ts-node": "^8.0.3",
+		"typescript": "^3.4.3",
 		"xo": "^0.24.0"
+	},
+	"ava": {
+		"babel": false,
+		"compileEnhancements": false,
+		"extensions": [
+			"ts"
+		],
+		"require": [
+			"ts-node/register"
+		]
+	},
+	"xo": {
+		"extends": "xo-typescript",
+		"extensions": [
+			"ts"
+		],
+		"rules": {
+			"@typescript-eslint/explicit-function-return-type": "off",
+			"typedef": [
+				true,
+				"call-signature",
+				"property-declaration",
+				"member-variable-declaration"
+			]
```

Comment:
Can you explain these rule changes?

Reply:
Also due to a rule in the old `tslint` config, it required 
```ts
() => {}
```

to have a defined return type but had the conflicting rule of `no redundant typedefs` since it could be inferred. Good we dropped that anyways 🔧